_miasta/wroclaw.html

```diff
@@ -0,0 +1,19 @@
+---
+layout: page
+title: Wrocław
+url-path: /miasta/wroclaw.html
+banner: wroclaw
+description: 10-11 września 2016, Uniwersytet Ekonomiczny
+map:
+  lat: 51.088525
+  lng: 17.025622
+  zoom: 13
+  icon: /images/d4k-46x46.png
+---
+<p>Wydarzenie jest skierowane dla dzieci w wieku od 6 do 15 lat. Dla każdego z czterech przedziałów wiekowych przygotowana jest oddzielna ścieżka. Przeprowadzane warsztaty wykorzystują zarówno darmowe jak i komercyjne rozwiązania dostępne na polskim rynku lub przez internet.</p>
+<ul class="actions vertical">
+    <li><a href="#" onclick="javascript:return false;" class="button fit special">Zapisy już wkrótce!</a></li>
```

Comment:
Ten kolor jest odcieniem koloru aktywnego przycisku i ma informowac o tym, ze przycisk jest nieaktywny. Mielismy taki przycisk na poznanskiej stronie zanim uruchomilismy rejestracje i dziwie sie takiej ocenie. Ale moze wiekszosc z nas nie zwrocila na to uwagi, albo tak jak ja nie ma za bardzo artystycznej duszy. 

Mozemy zmienic kolor nieaktywnego przycisku jesli masz jakies sugestie :-)